x-pack/test_user.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
**question** is this file being referenced or used anywhere?

Reply:
not being referenced anywhere. removing.